src/util/types.ts

```diff
@@ -1145,14 +1153,20 @@ export const filterProps = (
 
   const out: Record<string, any> = {};
 
+  /**
+   * If the svg element type is explicitly included, check against the filtered element key map
+   * to determine if there are attributes that should only exist on that element type.
+   * @todo Add an internal cjs version of https://github.com/wooorm/svg-element-attributes for full coverage.
```

Comment:
Could you please create an issue to follow up with this todo? Please link this PR to provide context for the issue.

Reply:
created and linked